hivemq-edge/src/frontend/.env

```diff
@@ -6,3 +6,6 @@ VITE_FLAG_MOCK_SERVER=false
 VITE_FLAG_FACET_SEARCH=true
 VITE_FLAG_WORKSPACE_FLOW_PANEL=true
 VITE_FLAG_TOPIC_EDITOR_SHOW_BRANCHES=false
+
+### User Analytics
+VITE_MONITORING_HEAP=3411251519
```

Comment:
Is it a value that we're comfortable with being present in the repo?

Reply:
Yes, its the default for the production tracker